src/custom-vocabularies-client.ts

```diff
@@ -0,0 +1,68 @@
+import { ApiRequestHandler } from './api-request-handler';
+import { CustomVocabulary } from './models/CustomVocabulary';
+import { CustomVocabularyOptions } from './models/CustomVocabularyOptions';
+import { RevAiApiCustomVocabulary } from './models/RevAiApiCustomVocabulary';
+
+/**
+* Client to submit and retreive status of custom vocabularies
+* from the rev.ai api. Check rev.ai/docs for more information.
```

Comment:
Much better :). Maybe we can also link to the specific section of the docs about custom vocabs

Reply:
When those are up for sure :)!